diff --git a/src/Plugin/DevelGenerate/WebformSubmissionDevelGenerate.php b/src/Plugin/DevelGenerate/WebformSubmissionDevelGenerate.php
index c072ae2..077ab36 100644
--- a/src/Plugin/DevelGenerate/WebformSubmissionDevelGenerate.php
+++ b/src/Plugin/DevelGenerate/WebformSubmissionDevelGenerate.php
@@ -52,13 +52,6 @@ class WebformSubmissionDevelGenerate extends DevelGenerateBase implements Contai
    */
   protected $database;

-  /**
-   * The entity type manager.
-   *
-   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
-   */
-  protected $entityTypeManager;
-
   /**
    * The webform submission generation service.
    *
@@ -90,7 +83,8 @@ class WebformSubmissionDevelGenerate extends DevelGenerateBase implements Contai
   /**
    * {@inheritdoc}
    */
-  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition): static
+  {
     $instance = new static($configuration, $plugin_id, $plugin_definition);
     $instance->request = $container->get('request_stack')->getCurrentRequest();
     $instance->database = $container->get('database');
@@ -105,7 +99,8 @@ public static function create(ContainerInterface $container, array $configuratio
   /**
    * {@inheritdoc}
    */
-  public function settingsForm(array $form, FormStateInterface $form_state) {
+  public function settingsForm(array $form, FormStateInterface $form_state): array
+  {
     $form['message'] = [
       '#type' => 'webform_message',
       '#message_message' => $this->t('Please note that no emails will be sent while generating webform submissions.'),
@@ -230,7 +225,8 @@ public function validateForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function generateElements(array $values) {
+  public function generateElements(array $values): void
+  {
     $this->generateSubmissions($values);
   }

@@ -348,7 +344,8 @@ protected function generateSubmission(&$results) {
   /**
    * {@inheritdoc}
    */
-  public function validateDrushParams(array $args, array $options = []) {
+  public function validateDrushParams(array $args, array $options = []): array
+  {
     $webform_id = array_shift($args);
     $webform_ids = [$webform_id => $webform_id];
     $values = [
